client/src/components/Form/Form.scss

```diff
@@ -171,7 +171,6 @@ input[type="checkbox"],
 input[type="radio"],
 input.checkbox,
 input.radio {
-  display: inline;
```

Comment:
This does nothing in most cases due to the usage of `float` - but it was causing problems with the checkbox in userforms submissions area.